qa/L0_model_config/test.sh

```diff
@@ -45,7 +45,8 @@ source ../common/util.sh
 
 export CUDA_VISIBLE_DEVICES=0
 
-TRIALS="tensorflow_savedmodel tensorflow_graphdef tensorrt_plan onnxruntime_onnx pytorch_libtorch custom"
+# Removing custom backend since legacy custom backend is deprecated
```

Comment:
Don't need this comment. No point of mentioning legacy backend after it is deprecated